extensions/aproc/proc/ingest/drivers/impl/image_driver_helper.py

```diff
@@ -42,32 +43,38 @@ def add_overview_if_you_can(driver: AbstractDriver, url: str, role: Role, size:
                           description=role.value, asset_format=AssetFormat.png.value)
             asset.href = driver.get_asset_filepath(url, asset)
             driver.LOGGER.debug("Try to create the thumbnail of {} in {}".format(url, asset.href))
-            image = Image.open(url)
+            local_url = AccessManager.prepare(url)
+            image = Image.open(local_url)
             image.thumbnail([size, size])
             image.save(asset.href, 'PNG')
-            asset.size = get_file_size(asset.href)
+            asset.size = AccessManager.get_file_size(asset.href)
             to_assets.append(asset)
             image.close()
+
+            if local_url != url:
+                os.remove(local_url)
```

Comment:
The system should never delete any file. If it is really needed, then this must be documented and we must use the comment "!DELETE!".

https://docs.google.com/document/d/1UTm5JUZ8F9ptlcya9v88hzg-iPWxeuCCziAD_oycloo/edit?tab=t.0